hasher-matcher-actioner/hmalib/tests/test_pdq_models.py

```diff
@@ -387,3 +388,44 @@ def test_writes_inc_counters_only_for_the_updated_class(self):
             record.write_to_table(self.table)
 
             assert 0 == models.PDQMatchRecord.get_total_count(self.table)
+
+
+class MatchByPrivacyGroupCounterTestCase(CountersTest):
```

Comment:
[Diff 2/5] nit: this test doesn't strike me as pdq specific, add a TODO note to refactor?

Reply:
https://github.com/facebook/ThreatExchange/issues/594#issue-894714795

Does this work? Haha. Went to file an issue. Saw you'd already done that. Resolved.